src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a variable for the radius value and pass it to the constructor for better code readability.